quarkus-integration/quarkus/deployment/src/main/java/ai/timefold/solver/quarkus/deployment/config/SolverBuildTimeConfig.java

```diff
@@ -60,6 +60,11 @@ public interface SolverBuildTimeConfig {
      */
     TerminationRuntimeConfig termination();
 
+    /**
+     * Enable the Nearby Selection quick configuration.
+     */
+    Optional<String> nearbyDistanceMeterClass();
```

Comment:
Shouldn't this return `Class<? extends NearbyDistanceMeter>` directly? The other properties also don't return `String`.

Reply:
Yes, using the type `Class` instead of `String` is possible.